pallets/runtime/develop/src/runtime.rs

```diff
@@ -803,6 +803,9 @@ construct_runtime!(
         // Must be before session.
         Babe: pallet_babe::{Module, Call, Storage, Config, Inherent, ValidateUnsigned} = 1,
 
+        // Initialise identity earlier as other configs depend on it
+        Identity: identity::{Module, Call, Storage, Event<T>, Config<T>} = 30,
```

Comment:
It could be better to have it after the timestamp pallet. We are using the timestamp pallet in the Identity config.